config/element-api.php

```diff
@@ -367,6 +367,135 @@ function getResearch($locale, $type = false)
     ];
 }
 
+/**
+ * API Endpoint: Get publication
+ * Get full details of all publications
+ */
+function getPublication($locale, $programmeSlug, $pageSlug = null)
+{
+    normaliseCacheHeaders();
+
+    $criteria = [
+        'site' => $locale,
+        'section' => 'publications',
+        'status' => EntryHelpers::getVersionStatuses()
+    ];
+
+    $isSingle = $pageSlug ? true : false;
+    if ($pageSlug) {
+        $criteria['slug'] = $pageSlug;
+    }
+
+    $associatedProgramme = Entry::find()->section(['fundingProgrammes', 'strategicProgrammes'])->status(['live', 'expired'])->slug($programmeSlug)->site($locale)->one();
+
+    $sortParam = \Craft::$app->request->getParam('sort');
```

Comment:
This sort logic is the same as the research docs right? Worth us moving this to a helper method?